MagickCore/stream.c

```diff
@@ -1290,8 +1290,15 @@ MagickExport Image *StreamImage(const ImageInfo *image_info,
   assert(stream_info->signature == MagickCoreSignature);
   assert(exception != (ExceptionInfo *) NULL);
   read_info=CloneImageInfo(image_info);
+  if (read_info == (ImageInfo *) NULL)
+    return (Image *) NULL;
   stream_info->image_info=image_info;
   stream_info->quantum_info=AcquireQuantumInfo(image_info,(Image *) NULL);
+  if (stream_info->quantum_info == (QuantumInfo *) NULL)
```

Comment:
`AcquireQuantumInfo` will throw a fatal exception when it fails so we also don't need this check.

Reply:
My bad!! We need this check it is possible that it returns null!

Reply:
@jgj212 Can you restore this check. I did not check this correctly. It is possible that `AcquireQuantumInfo` returns null.